orttraining/orttraining/test/python/orttraining_run_glue.py

```diff
@@ -0,0 +1,216 @@
+# Code ported from run_glue.py of huggingface transformers
+# Still missing gradient clipping and the approach in AdamW.
+# Finetuning sequence classification models on GLUE (Bert, XLM, XLNet, RoBERTa, Albert, XLM-RoBERTa).
+
+import dataclasses
+import logging
+import os
+from dataclasses import dataclass, field
+from typing import Dict, Optional
+import unittest
+import numpy as np
+
+from transformers import (
+    AutoConfig,
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+    EvalPrediction,
+    GlueDataset,
+    GlueDataTrainingArguments,
+    TrainingArguments,
+    glue_compute_metrics,
+    glue_output_modes,
+    glue_tasks_num_labels,
+    set_seed,
+)
+
+import onnxruntime
+from onnxruntime.capi.ort_trainer import ORTTrainer, LossScaler, ModelDescription, IODescription
+
+from orttraining_transformer_trainer import ORTTransformerTrainer
+
+import torch
+
+logger = logging.getLogger(__name__)
+
+@dataclass
+class ModelArguments:
+    """
+    Arguments pertaining to which model/config/tokenizer we are going to fine-tune from.
+    """
+
+    model_name_or_path: str = field(
+        metadata={"help": "model identifier from huggingface.co/models"}
+    )
+    config_name: Optional[str] = field(
+        default=None, metadata={"help": "Pretrained config name or path if not the same as model_name"}
+    )
+    tokenizer_name: Optional[str] = field(
+        default=None, metadata={"help": "Pretrained tokenizer name or path if not the same as model_name"}
+    )
+    cache_dir: Optional[str] = field(
+        default=None, metadata={"help": "Where do you want to store the pretrained models downloaded from s3"}
+    )
+
+class ORTGlueTest(unittest.TestCase):
+
+    def setUp(self):
+        # configurations not to be changed accoss tests
+        self.max_seq_length = 128
+        self.train_batch_size = 8
+        self.learning_rate = 2e-5
+        self.num_train_epochs = 3.0
+        self.local_rank = -1
+        self.overwrite_output_dir = True
+        self.gradient_accumulation_steps = 1
+        self.data_dir = "/bert_data/hf_data/glue_data/"
+        self.output_dir = os.path.dirname(os.path.realpath(__file__)) + "/glue_test_output/"
```

Comment:
ideally paths should be joined through `os.path.join(path1, path2, path3, etc)` to ensure they are portable as opposed to hard-coding "/" on them